functions/getWorkerAttributes.ts

```diff
@@ -0,0 +1,58 @@
+import '@twilio-labs/serverless-runtime-types';
+import {
+  Context,
+  ServerlessCallback,
+  ServerlessFunctionSignature,
+} from '@twilio-labs/serverless-runtime-types/types';
+import {
+  responseWithCors,
+  bindResolve,
+  error400,
+  error500,
+  success,
+} from '@tech-matters/serverless-helpers';
+
+const TokenValidator = require('twilio-flex-token-validator').functionValidator;
+
+type EnvVars = {
+  TWILIO_WORKSPACE_SID: string;
+};
+
+export type Body = {
+  workerSid?: string;
+};
+
+export const handler: ServerlessFunctionSignature = TokenValidator(
+  async (context: Context<EnvVars>, event: Body, callback: ServerlessCallback) => {
+    const response = responseWithCors();
+    const resolve = bindResolve(callback)(response);
+
+    try {
+      const { workerSid } = event;
+
+      if (workerSid === undefined) {
+        resolve(error400('workerSid'));
+        return;
+      }
+
+      const client = context.getTwilioClient();
+      const worker = await client.taskrouter
+        .workspaces(context.TWILIO_WORKSPACE_SID)
+        .workers(workerSid)
+        .fetch();
+
+      const workerAttributes = JSON.parse(worker.attributes);
```

Comment:
I think this is too much of a security issue.  This gives any counselor, regardless of their permissions, access to all the attributes of another counselor.  In a large organization with hundreds of volunteers, that's way too much data.  Especially since we might add more attributes in the future and forget that they're coming through here.

Let's just whitelist the `helpline` attribute and return that only (but we should return it inside an object so that we can possibly add more attributes later.